tests/e2e/utils/workspace/TestWorkspaceUtil.ts

```diff
@@ -129,6 +130,39 @@ export class TestWorkspaceUtil implements ITestWorkspaceUtil {
         await this.deleteWorkspaceByName(namespace, workspaceName);
     }
 
+    // stop all run workspaces in the namespace
+    public async stopAllRunProjects(namespace: string) {
+        Logger.debug('TestWorkspaceUtil.stopAllRunProjects');
+        let response = await this.processRequestHandler.get(this.apiUrlResolver.getWorkspacesApiUrl(namespace));
+        for (let i = 0; i < response.data.items.length; i++) {
+            Logger.info('The project is being stopped: ' +  response.data.items[i].metadata.name);
+            await this.stopWorkspaceByName(namespace, response.data.items[i].metadata.name);
+        }
+    }
+
+    // stop all run workspaces, check statused and remove the workspaces
+    public async stopAndDeleteAllRunProjects(namespace: string) {
+        Logger.debug('TestWorkspaceUtil.stopAndDeleteAllRunProjects');
+        let response = await this.processRequestHandler.get(this.apiUrlResolver.getWorkspacesApiUrl(namespace));
+        await this.stopAllRunProjects(namespace);
+        for (let i = 0; i < response.data.items.length; i++) {
+            Logger.info('The project is being deleted: ' +  response.data.items[i].metadata.name);
+            await this.deleteWorkspaceByName(namespace, response.data.items[i].metadata.name);
+        }
+    }
+
+    // stop all run workspaces without stopping and waiting for of 'Stopped' phase
+    // similar with 'force' deleting
+    public async deleteAllRunProjects(namespace: string) {
```

Comment:
IMHO, `deleteAllWorkspaces()` method name is more descriptive, because the method is currently deleting workspaces, not projects, and it also deleting all stopped workspaces, not just running workspaces.